.eslintrc

```diff
@@ -3,13 +3,15 @@
     "ecmaVersion": 2017,
     "sourceType": "module"
   },
+  "parser": "babel-eslint",
   "extends": [
     "airbnb",
     "prettier",
     "eslint:recommended",
     "plugin:react/recommended",
     "plugin:jsx-a11y/recommended",
-    "plugin:import/recommended"
+    "plugin:import/recommended",
+    "react-app"
```

Comment:
I don't think this is needed - https://www.npmjs.com/package/eslint-config-react-app - says you shouldn't need it if you are already using create-react-app since it's the `react-scripts`.